src/lib/accountBalances.js

```diff
@@ -22,6 +23,20 @@ AccountBalances.prototype.applyDebits = function * () { yield this._applyDebits(
 AccountBalances.prototype.applyCredits = function * () { yield this._applyCredits(this._credits) }
 AccountBalances.prototype.revertDebits = function * () { yield this._applyCredits(this._debits) }
 
+function sum (numbers) {
+  const numStrings = _.map(numbers, String)
+  return _.reduce(_.rest(numStrings), (result, num) => {
```

Comment:
From the lodash docs: "If accumulator is not given the first element of collection is used as the initial value."

So you don't have to use `first` and `rest`, just pass `numStrings` as the first argument.